homeassistant/components/hassio/ingress.py

```diff
@@ -167,7 +167,14 @@ def _init_header(
 
     # filter flags
     for name, value in request.headers.items():
-        if name in (hdrs.CONTENT_LENGTH, hdrs.CONTENT_ENCODING):
+        if name in (
+            hdrs.CONTENT_LENGTH,
+            hdrs.CONTENT_ENCODING,
+            hdrs.SEC_WEBSOCKET_EXTENSIONS,
+            hdrs.SEC_WEBSOCKET_PROTOCOL,
```

Comment:
Are you sure you don't need to forward protocol? It's part of what we pass into the constructor in JS https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/WebSocket

Reply:
Yes, because we don't root WebSockets, we did create our own connection between the points and forward data. aiohttp will overwrite this value but I will be saved and don't attach it into headers.